### PR TITLE
Handle multiple people performing labeling.

### LINKED ISSUE #21: Modify label.py to handle multiple people performing the labeling
Modify the label.py script to handle multiple people performing the labeling (I would suggest simply adding a parameter PERSON, and writing a labels_PERSON.csv according to this parameter). Then, write another script that generates labels.csv from all the labels_*.csv files (basically storing the lines from labels_*.csv which agree on the label).

### PR DESCRIPTION
**Pull request by sagarvijaygupta**
Add UI to take input of filename.
generate_labels.py - creates labels.csv on basis of voting from different files.

Fixes #21

### CODE REVIEW

**Comment by marco-c** [comment]
Could you rebase on top of master? There are conflicts in label.py and utils.py.

File: generate_labels.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+labels_directory = "label_persons/"
+all_file_names = [f for f in listdir(labels_directory)]
+
+labels_voted = {}
+ydn_map = {'y': 0, 'd': 1, 'n': 2}
+ydn_reverse_map = {0: 'y', 1: 'd', 2: 'n'}
+
+for file in all_file_names:
```
Comment (marco-c) [inline generate_labels.py]: Can you use a different variable name here? `file` is a Python keyword so it gets highlighted differently than other variable (which can be confusing). Maybe use `file_name`

File: generate_labels.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    labels = read_labels(labels_directory + file)
+    for key, value in labels.items():
+        if key not in labels_voted.keys():
+            labels_voted[key] = [0, 0, 0]
+        labels_voted[key][ydn_map[value]] += 1
+
+with open('labels.csv', 'w') as f:
```
Comment (marco-c) [inline generate_labels.py]: Let's move file writing in utils.py.

File: label.py [hunk 3]
```diff
@@ -58,17 +57,50 @@ def close(e):
...
+        os.makedirs(labels_directory)
+    all_file_names = [f for f in os.listdir(labels_directory)]
+
+    frame = Frame(root)
+    label_file_name = StringVar(frame)
+    frame.pack(side="top")
+    input_type = Label(frame, text="Choose or add filename with .csv extension")
+    input_type.pack(side="top")
+    if len(all_file_names) != 0:
+        old_user = OptionMenu(frame, label_file_name, *all_file_names)
```
Comment (marco-c) [inline label.py]: The option menu is a little confusing, we should add some kind of placeholder.

Or, we can improve the form by doing something like:
Choose existing label file: [OPTIONMENU]
Create new label file: [TEXTINPUT]

Or, we can use argparse and pass the name of the label file as a parameter to the label.py script (so you would run it , for example, as `python label.py marco`) and remove this `choose_file` function.

All three options are fine by me, so feel free to decide which one you prefer.

**Comment by marco-c** [comment]
Another thing, since #23 was merged, please move the current labels.csv file under label_persons (with the name amit2rockon).

**Comment by sagarvijaygupta** [comment]
@marco-c ```labels.csv``` that was merged seems to be a ```json``` file with its extension as ```csv```.

**Comment by marco-c** [comment]
> @marco-c labels.csv that was merged seems to be a json file with its extension as csv.

Thanks for noticing, I will revert the PR.

**Comment by marco-c** [comment]
There's a conflict because I've reverted the PR which was adding the file in the wrong format.

**Comment by sagarvijaygupta** [comment]
@marco-c please review!

File: label.py [hunk 4]
```diff
@@ -1,11 +1,17 @@
-import csv
 from PIL import ImageTk, Image
 from tkinter import Tk, Label
+import argparse
```
Comment (marco-c) [inline label.py]: Nit: move this at the top. For imports we're using the following ordering:
1. Built-in modules first
2. Modules from third-parties
3. Our modules (like `utils`)

Comment (sagarvijaygupta) [reply label.py]: Done!

**Pull request merged**